src/qthotkey/meson.build

```diff
@@ -1,17 +1,19 @@
+x11_dep = dependency('x11-xcb', required: false)
```

Comment:
Nice approach, it's better than relying on `host_machine.system()`. :+1:
I would check for `x11` though. Any reason to use `x11-xcb` here?

Reply:
Firstly, qthotkey [has an import from xcb](https://github.com/audacious-media-player/audacious-plugins/blob/audacious-plugins-4.3/src/qthotkey/plugin.cc#L59).  
Secondly, Qt [implements](https://doc.qt.io/qt-6/linux-requirements.html) X11 compatibility through xcb.

Reply:
But x11-xcb was still found on macOS, so it seems `host_machine.system()` is inevitable.